llvm/test/Transforms/Coroutines/ArgAddr.ll

```diff
@@ -5,7 +5,7 @@
 define nonnull ptr @f(i32 %n) presplitcoroutine {
 ; CHECK-LABEL: @f(
 ; CHECK-NEXT:  entry:
-; CHECK-NEXT:    [[ID:%.*]] = call token @llvm.coro.id(i32 0, ptr null, ptr null, ptr @f.resumers)
+; CHECK-NEXT:    [[ID:%.*]] = call token @llvm.coro.id(i32 0, ptr null, ptr null, ptr @{{.*}})
```

Comment:
Are these changes to `@llvm.coro.id` related to this patch really?

Reply:
Ah, the `noalloc` variant was appended to `f.resumers`, causing the name to be changed to `f.resumers.1`.

Reply:
I feel this is something need to be fixed. It looks odd to treat the  noalloc variant as resumers.

Reply:
Other than the legacy `CoroElide`, the main purpose of this variable is to keep connection on the graph between the original copy and the `.resume`, `.destroy`, `.cleanup`, `.noalloc` splits. I would suggest just changing the name to read "f.splits" instead of `f.resumers`. It's a requirement for CGSCC call graph updating.